kie-plugins-testing/src/test/java/org/kie/maven/plugin/KieMavenPluginBaseIntegrationTest.java

```diff
@@ -54,20 +59,39 @@ protected void restoreKiePom(String projectName) throws Exception {
     }
 
     protected MavenExecutionResult buildKJarProject(String kjarName, String[] options, String... mavenGoals) throws Exception {
+        logger.info("buildKJarProject {} {} {}", kjarName, options, mavenGoals);
         final MavenRuntime mavenRuntime = mavenRuntimeBuilder
```

Comment:
Is it possible to move this code inside the if block to avoid to initialize the runtime if the value is already available?

Reply:
@danielezonca 
Nope